### PR TITLE
Integration test and example app

### LINKED ISSUE #16: Add vue3 integration test
## Description

Add some minimal automated test that makes sure that the library works in vue3.
The test should be part of the CI.

## Tasks

- [ ] Create a small sample project with vue 3
- [ ] Adds minimal tests, maybe some basic component interaction that verifies that the library component loads and displays properly

### PR DESCRIPTION
**Pull request by tewshi**
Closes #16

### CODE REVIEW

File: example/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (kelsos) [inline example/cypress/fixtures/example.json:5]: is this used in the final thing?

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "vue-router": "4.2.2"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "2.0.1",
+    "@types/node": "18.16.8",
+    "@vitejs/plugin-vue": "4.2.3",
+    "@vue/tsconfig": "0.4.0",
+    "autoprefixer": "10.4.14",
+    "cypress": "12.13.0",
+    "npm-run-all": "4.1.5",
```
Comment (kelsos) [inline example/package.json:29]: Out of curiosity what command does this provide?

File: example/cypress/fixtures/example.json (hunk 1 above)
Comment (tewshi) [reply example/cypress/fixtures/example.json:5]: basic scaffold on cypress adds this example, will remove it

File: example/package.json (hunk 2 above)
Comment (tewshi) [reply example/package.json:29]: haha, i'll remove it, but it enables running multiple commands

like `pnpm build && pnpm start` as `run-p build start`

Comment (kelsos) [reply example/package.json:29]: No worries it's ok, it was just not obvious. no need to remove it if it is used.

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**